examples/tutorials/forhowto/plot_1_working_with_tetrodes.py

```diff
@@ -0,0 +1,111 @@
+
+##############################################################################
+# In practice, you would read in your raw data from a file. Instead, we will generate a
+# recording with eight channels. We can also set a duration, number of units and
+# sampling frequency.
+
+recording, _ = generate_ground_truth_recording(
+    durations = [60], # make the recording 60s long
+    sampling_frequency=30_000,
+    num_channels=8,
+    num_units=10,
+)
+
+##############################################################################
+# We now need to define the probe. This will tell the recording which channels came from
+# which tetrode. To do this, we will use the :code:`generate_tetrode` function from :code:`ProbeInterface`
+# to generate two 4-channel probes (representing one tetrode each). In our case, since we
+# don't know the relative distances between the tetrodes, we will move the second
+# tetrode away from the first by 100 microns. This is just so we can visualize the
+# results more easily. Eventually, we will sort each tetrode separately, so their
+# relative distance won't affect the results.
+
+# Technically, we will add each tetrode to a :code:`ProbeGroup`. Read more in the ProbeInterface
+# docs.
+
+# Create each individual tetrode
+tetrode_1 = generate_tetrode()
+tetrode_1.create_auto_shape()
+
+tetrode_2 = generate_tetrode()
+tetrode_2.move([100, 0])
+tetrode_2.create_auto_shape()
+
+# Add the two tetrodes to a ProbeGroup
+tetrode_group = ProbeGroup()
+tetrode_group.add_probe(tetrode_1)
+tetrode_group.add_probe(tetrode_2)
+
+# Now we need to "wire" our tetrodes to ensure that each contact
+# can be associated with the correct channel when we attach it
+# to the recording. In this example we are just using `range`
+# but see ProbeInterface for more tutorials on wiring
+tetrode_group.set_global_device_channel_indices(range(8))
+
+##############################################################################
+# We can now attach the :code:`tetrode_group` to our recording. To check if this worked, we'll
+# plot the probe map
+
+recording_with_probe = recording.set_probegroup(tetrode_group)
+plot_probe_map(recording_with_probe)
+
+##############################################################################
+# Looks good! Now that the recording is aware of the probe geometry, we can
+# begin a standard spike sorting pipeline. First, we can apply preprocessing.
+# Note that we apply this preprocessing on the entire bundle of tetrodes.
+# This is appropriate for a common reference because the tetrodes
+# share the same background noise.
+
+preprocessed_recording = spre.bandpass_filter(
+    spre.common_reference(
+        recording_with_probe
+    )
+)
+
+##############################################################################
+# Tetrodes often have dead channels, so it is advised to try and detect
+# and remove these
+
+recording_good_channels = spre.detect_and_remove_bad_channels(preprocessed_recording)
```

Comment:
the default `coherence+psd` method uses depth information at some steps and it assumes a "linear" gometry. We should use the `std`/`mad` mode, that just looks for too high noise levels